src/Our.Umbraco.StackedContent/Web/UI/App_Plugins/StackedContent/views/stackedcontent.html

```diff
@@ -4,9 +4,10 @@
 
     <div ng-show="inited">
 
-        <a ng-if="!model.value || model.value.length == 0" ng-click="addContent($event, 0)" class="placeholder" title="Add content">
+        <div ng-if="!model.value || model.value.length == 0"ng-click="addContent($event, 0)" class="placeholder"  title="Add content">
```

Comment:
Something is wrong here. This should be two links side by side, but it appears the paste link is inside the add link. Needs refactoring.

Reply:
I'd played around with various options - tried 2 `<a>` side-by-side, we'd lose the functionality to click anywhere inside the "Add content" panel - users would have to click the "icon-umb-contour" icon.